### ISSUE TITLE
Exposecontroller does not run and stuck at CrashLoopBackOff

### ISSUE THREAD

**Issue by noorulhaq** [fabric8io/fabric8#6883]
Hi, 

I am getting below error when I use `gofabric8 deploy` command. Note I have not specified any `--domain` argument here. 

```
I0619 10:25:13.284013       1 exposecontroller.go:47] Using build: '2.3.2'
F0619 10:25:13.389317       1 exposecontroller.go:66] failed to create new strategy: failed to create ingress expose strategy: failed to get a domain: failed to find any nodes: the server does not allow access to the requested resource (get nodes)
```

**Comment by jstrachan** [comment]
I'm guessing you are running on kubernetes? If so you need to specify a domain to reuse the Ingress strategy

**Comment by noorulhaq** [comment]
Yes, I am trying to deploy it on existing Kubernetes cluster. Does that mean I have to have a public domain name if I am using a machine on cloud?

**Comment by noorulhaq** [comment]
Replication controller now started, but I ran into another problem:

Now the system is stuck at below state:

```
Checking if PersistentVolumeClaims bind to a PersistentVolume 
Downloading images and waiting to open the fabric8 console...

-------------------------
.........................................................
```

**Comment by chrisvugrinec** [comment]
I am running Kubernetes on Azure: same problem here..

I got unstuck by specifying my public ip address of my kube master
gofabric8 deploy --domain [ my domain name here] 
for eg
I did:
gofabric8 deploy --domain vuggiedemo1.northeurope.cloudapp.azure.com
then it tried to open 
https://fabric8.default.vuggiedemo1.northeurope.cloudapp.azure.com/
but nothing happened: all pods are up...do I need to open additional ports?

**Comment by akshayjain1996** [comment]
Running locally using minikube and I run into the same problem. I tried to increase memory and cpu, but no luck
